src/lib/ui/layout/DetailSection.tsx

```diff
@@ -15,8 +15,8 @@ export function DetailSection({
   return (
     <div className='seam-detail-section'>
       <div className='seam-detail-label-wrap'>
-        <p className='seam-detail-label'>{label}</p>
-        {tooltipContent != null && <Tooltip>{tooltipContent}</Tooltip>}
+        {label !== undefined && <p className='seam-detail-label'>{label}</p>}
+        {tooltipContent !== undefined && <Tooltip>{tooltipContent}</Tooltip>}
```

Comment:
These would be undefined if not provided, no?

Reply:
Prefer to compare to null like this unless there is a reason to differentiate between null and undefined. 
```suggestion
        {label != null && <p className='seam-detail-label'>{label}</p>}
        {tooltipContent != null && <Tooltip>{tooltipContent}</Tooltip>}
```

Reply:
<img width="399" alt="Screenshot 2023-08-31 at 1 09 48 PM" src="https://github.com/seamapi/react/assets/11382084/97dc49c6-8f26-4204-b853-0995400553fc">

These pop up as undefined for me though. Does `null` also work here?